product-service/handler.js

```diff
@@ -0,0 +1,4 @@
+import { getProductsList } from './handlers/getProductsList.js';
```

Comment:
you can export them like that:
`export * from './handlers/getProductsList.js';`